packages/create-request-form/src/lib/create-request-form.svelte

```diff
@@ -0,0 +1,126 @@
+<script lang="ts" customElements="create-request-form">
+  import { ethers } from 'ethers';
+  import { Types, Utils } from '@requestnetwork/request-client.js';
+  import type { RequestNetwork } from '@requestnetwork/request-client.js';
+
+  export let requestNetwork: RequestNetwork;
+  export let signer: string; 
+  let amount = '';
+  let payerAddress = '';
+  let payeeAddress = '';
+  let paymentReason = 'Service Payment';
+  let dueDate = '';
+  let requestInitialized = false;
+  
+  async function submitForm() {
+    if (!requestNetwork || !signer) {
+      console.error('Request Network or signer not properly initialized.');
+      return;
+    }
+
+    const requestCreateParameters = {
+      requestInfo: {
+        currency: {
+          type: Types.RequestLogic.CURRENCY.ETH,
+          value: 'ETH',
+          network: 'sepolia'
+        },
+        expectedAmount: ethers.utils.parseUnits(amount, 'ether').toString(),
```

Comment:
Ethers v5 has a special function for Ether parsing.

```suggestion
        expectedAmount: ethers.utils.parseEther(amount).toString(),
```